### ISSUE TITLE
Inlezen tarieven van andere leveranciers

### ISSUE THREAD

**Issue by Lamkram** [gruijter/com.gruijter.powerhour#262]
Hallo,

Geen probleem maar een vraag. Kan je ook andere tarieven inlezen met de PBTH? Ik neem namelijk stroom af van zonneplan en die tarieven zijn anders dan bv energy zero.

Hoor graag van je.

Groet en dank,

Mark

**Comment by gruijter** [comment]
nope. you can set any NL provider by tweaking the DAp settings. see homey forum https://community.homey.app/t/app-pro-power-by-the-hour-insights-per-hour-day-month-and-year/20535
